actions/test-dependency/dependency-tests/poetry/run

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -euo pipefail
+
+extract_tarball_poetry() {
+  rm -rf poetry
+  mkdir poetry
+  tar -xf dependency/*.tgz -C poetry --wildcards ./poetry*.tar.gz
+  tar -xf poetry/poetry*.tar.gz -C poetry
+}
+
+extract_tarball_python() {
+  rm -rf python
+  mkdir python
+  tar -xf required_dependency/*.tgz -C python
+}
+
+set_ld_library_path() {
+  export LD_LIBRARY_PATH="$PWD/python/lib:${LD_LIBRARY_PATH:-}"
+}
+
+check_version() {
+  expected_version="$1"
+  actual_version="$(./python/bin/python ./poetry/poetry-*/setup.py --version)"
```

Comment:
@thitch97 I haven't checked the output of the built version of poetry, but can you validate that `setup.py` is still a valid script to call? I know it's what we did in `pip` but i don't know if it's a pip-specific file or if we can expect it to be present in poetry too.

Reply:
Yes, it's standard for python packages and I validated that it works locally

Reply:
awesome - thanks!